src/gauging.jl

```diff
@@ -74,21 +70,21 @@ function vidal_gauge(
   return ψ_vidal, bond_tensors
 end
 
-"""Use an ITensorNetwork ψ and its mts to put ψ into the vidal gauge, bond tensors are initialised and made empty. Return the bond tensors and ψ_vidal."""
+"""Use an ITensorNetwork ψ in the symmetric gauge and its mts to put ψ into the vidal gauge. Return the bond tensors and ψ_vidal."""
 function vidal_gauge(
   ψ::ITensorNetwork,
   mts::DataGraph;
   eigen_message_tensor_cutoff=10 * eps(real(scalartype(ψ))),
   regularization=10 * eps(real(scalartype(ψ))),
   svd_kwargs...,
 )
-  bond_tensors = DataGraph{vertextype(ψ),ITensor,ITensor}(underlying_graph(ψ))
+  bond_tensors = intialise_bond_tensors(ψ)
```

Comment:
`intialise` to `initialize`.

Reply:
Apparently "initialise" is the British spelling, I didn't know that! https://english.stackexchange.com/questions/459553/initialised-or-initialized-which-one-is-correct-spelling.

Reply:
Yeah it is! Similarly with colour vs color and neighbour vs neighbor. Makes sense to use the American spelling though - might confuse people otherwise. Just think I am on autopilot when I name things